index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MeowFeed</title>
+    <link rel="stylesheet" href="dist/css/index.css" />
+  </head>
+  <body>
+    <div
+      class="container min-vh-100 d-flex justify-content-center align-items-center"
+      style="max-width: 800px"
```

Comment:
Use Bootstrap breakpoint classes instead

Reply:
removed style=max-width and fixed the issue, the issue was in <img>